fastlane/Fastfile

```diff
@@ -48,6 +48,9 @@ ENV["DOWNLOAD_METADATA"]="fastlane/download_metadata.swift"
 ENV["PROJECT_ROOT_FOLDER"]="./"
 ENV["APP_STORE_STRINGS_FILE_NAME"]="AppStoreStrings.pot"
 
+# Other defines used across multiple lanes
+REPOSITORY_NAME="WordPress-Android"
```

Comment:
`woocommerce-ios`

Reply:
Good catch. 
There are still a few things to tweak and test. I plan to finish this and to put it out of the "draft" state tomorrow 😅